src/quic/node_quic_socket.h

```diff
@@ -166,7 +166,8 @@ class QuicPacket : public MemoryRetainer {
   SET_SELF_SIZE(QuicPacket);
 
  private:
-  std::vector<uint8_t> data_;
+  uint8_t data_[NGTCP2_MAX_PKTLEN_IPV4];
```

Comment:
The PR message talks about using `NGTCP2_MAX_PKT_SIZE` but new code uses `NGTCP2_MAX_PKTLEN_IPV4` is that correct? (note that the check in constructor `QuicPacket::QuicPacket(const char* diagnostic_label, size_t len)` still checks for `NGTCP2_MAX_PKT_SIZE`)

Also, `NGTCP2_MAX_PKT_SIZE` recently changed to `NGTCP2_DEFAULT_MAX_UDP_PAYLOAD_SIZE` (https://github.com/ngtcp2/ngtcp2/commit/38feb91ee1e0556696434556923bcff7093ea6cd)

Reply:
I'll double check on that. May have mentally mixed those up